src/tools/pub/cmd/pub-add.md

```diff
@@ -9,10 +9,10 @@ _Add_ is one of the commands of the [pub tool](/tools/pub/cmd).
 $ dart pub add <package>[:<constraint>] [<package2>[:<constraint2>]... ] [options]
 ```
 
-This command adds the specified packages to the pubspec as dependencies, 
-and then gets the dependencies.
+This command adds the specified packages to the `pubspec.yaml` as dependencies, 
+and then gets the dependencies.  
```

Comment:
Retrieves the dependencies?

Reply:
Maybe even
```suggestion
and then resolves the pubspec.yaml.  
```

Reply:
I tried combining both suggestions
```
This command adds the specified packages to the `pubspec.yaml` as dependencies, 
and then retrieves the dependencies to resolve `pubspec.yaml`.  
```
Let me know if that doesn't make sense!